Sources/AccountSDKIOSWeb/Lib/MFAType.swift

```diff
@@ -13,7 +13,7 @@ public enum MFAType: Codable {
     /// BankId verification for PRE environment
     case preEid(PreEidType)
 
-  var rawValue: String {
+  public var rawValue: String {
```

Comment:
Why was this made public?

Reply:
I use it to set acr value in additional fields (for pre env there is another type PreEidType). Let's try if I can do it without making it explicitly public